index.bs

```diff
@@ -254,9 +254,14 @@ Non-Functional Requirements {#requirements-non-functional}
 Discovery with mDNS {#discovery}
 ===============================
 
-[=Open Screen Protocol agents=] must discover one another using
-[[RFC6763|DNS-SD]] over [[RFC6762|mDNS]].  To do so, OSP agents must use the
-[=Service Name=] `_openscreen._udp.local`.
+[=Open Screen Protocol agents=] discover one another by advertising and
+listening for information identifying themselves along with an IP service
+endpoint.  Agent advertisment and discovery through [[RFC6763|DNS-SD]] and
+[[RFC6762|mDNS]] is defined by this specification and is mandatory to implement
+by all agents. However, agents are free to implement additional discovery
+mechanisms, such as querying for the same DNS-SD records via unicast DNS.
+
+OSP agents must use the DNS-SD [=Service Name=] `_openscreen._udp.local`.
```

Comment:
The Service Name is `_openscreen._udp` for both DNSSD and mDNS. The next component is the Domain, and that's `local` in mDNS and can be anything else in DNSSD. I'd suggest changing this instance and others in this file. It's good for `local` to show up in examples but I'd avoid it in normative text because that effectively disallows unicast DNS.

```suggestion
OSP agents must use the DNS-SD [=Service Name=] `_openscreen._udp`.
```

Reply:
Since mDNS is mandatory and unicast DNS is additional/optional maybe we can make it clearer.   

_OSP agents must use the DNS-SD [=Service Name=] `_openscreen._udp.local` for mDNS and `_openscreen._udp.*` if unicast DNS is supported._ 

Maybe the `*` in  `_openscreen._udp.*` needs to be replaced with something concrete? wdyt?

Reply:
@louaybassbouss sorry to be slightly pedantic here, but `_openscreen._udp.local` is not a service name. In that example, `_openscreen._udp` is the service name and `local` is the domain. The term "service name" is defined in [RFC 6763 Section 4.1](https://datatracker.ietf.org/doc/html/rfc6763#section-4.1).

Reply:
@DavidSchinazi thanks for clarification LGTM.